packages/fxa-react/components/Head/index.tsx

```diff
@@ -14,19 +14,25 @@ const localeDirection = determineDirection(supportedUserLocale);
 
 const Head = ({ title }: { title?: string }) => {
   const { l10n } = useLocalization();
+
+  const customTitle = title
+    ? l10n.getString(
+        'app-page-title-2',
+        { title },
+        `${title} | Mozilla accounts`
+      )
+    : l10n.getString('app-default-title-2', null, 'Mozilla accounts');
+
+  // setting the document title here ensures it gets picked up by Glean automatic page load metrics
+  useEffect(() => {
+    document.title = customTitle;
```

Comment:
Question: Wouldn't this set a localized title that Glean automatically picks up (which is what we don't want)?

Reply:
It does - but the only way I've found to override is to manually trigger the glean page load events on each page with an override for the document title - the compromise here was to use the pathname instead of the title for data grouping.

In re-testing though I've found that the correct title is only occasionally included in the glean ping, seems the page load event is sometimes (but not always) triggered before the title is defined... Looking into this more.

Reply:
To wrap up - I looked into manually triggering the page load events closer to the render. This worked well for the Settings pages, but the other pages were a bit of a rabbit hole with lots of tests impacted and requiring mock updates. I opted to include the Settings update (custom titles should now be included correctly, although localized), but to skip the manual triggering for other pages considering that most of these don't have custom document titles anyway (see https://mozilla-hub.atlassian.net/browse/FXA-6803). All page load events will however now include the pathname instead of the full URL, which should resolve the main concern for this ticket.